pandas/io/tests/test_pytables.py

```diff
@@ -4743,6 +4743,170 @@ def test_read_nokey(self):
             df.to_hdf(path, 'df2', mode='a')
             self.assertRaises(ValueError, read_hdf, path)
 
+class TestHDFComplexValues(tm.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
can you create a ``Base(tm.TestCase)`` to put these (and have ``TestHDFStore`` inherit as well), so hide the boilerplate a bit